apps/bajour/src/components/bajour/archive/archive.tsx

```diff
@@ -1,32 +1,27 @@
-import {Button, styled} from '@mui/material'
+import {css, styled} from '@mui/material'
 import {
-  ApiV1,
   BuilderTeaserGridBlockProps,
   selectTeaserAuthors,
   selectTeaserLead,
   selectTeaserTitle,
   selectTeaserUrl,
   useWebsiteBuilder
 } from '@wepublish/website'
-import Image from 'next/image'
 import {useState} from 'react'
 
-import Logo from '../../../logo.svg'
+import {ReactComponent as Logo} from '../../../logo.svg'
 import {ArchiveSlider} from './archive-slider'
 
-export type ArchiveProps = Omit<BuilderTeaserGridBlockProps, 'teasers'> & {
-  teasers: ApiV1.ArticleTeaser[] | ApiV1.PeerArticleTeaser[]
```

Comment:
We don't know the type of teaser. Using teaser functions means we can also show pages or events if wanted